chart/templates/deployment.yaml

```diff
@@ -0,0 +1,151 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "obot.fullname" . }}
+  labels:
+    {{- include "obot.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "obot.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "obot.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "obot.serviceAccountName" . }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            exec:
+              command:
+                - "true"
+          readinessProbe:
+            exec:
+              command:
+                - "true"
```

Comment:
Doesn't need to be fixed right away, but I think there is probably some route we could hit to check server readiness. And if not, that's something we could add to the backend.

Reply:
@thedadams said he would add an endpoint for me